### PR TITLE
Developer

### PR DESCRIPTION
**Pull request by elena-krismer**

### CODE REVIEW

**Comment by elena-krismer** [comment]
Screenshot 2022-08-26 at 14 42 59
my bad github practices got me a github achievement

File: alphastats/loader/MaxQuantLoader.py [hunk 1]
```diff
@@ -33,15 +33,15 @@ def __init__(
...
-            self.load_evidence(evidence_file=evidence_file)
+            self._load_evidence(evidence_file=evidence_file)
 
-    def load_evidence(self, evidence_file, sep = "\t"):
+    def _load_evidence(self, evidence_file, sep="\t"):
         self.evidence_file = pd.read_csv(evidence_file, sep=sep, low_memory=False)
         # check if names match protien group file
 
-    def set_filter_columns_to_true_false(self):
+    def _set_filter_columns_to_true_false(self):
```
Comment (straussmaximilian) [inline alphastats/loader/MaxQuantLoader.py:44]: `to_true_false` is a bit hard to understand? Maybe something like convert_to_bool?

File: setup.py [hunk 2]
```diff
@@ -81,14 +77,12 @@ def create_pip_wheel():
...
-            "console_scripts": "alphastats=alphastats.cli:run",
-        },
+        entry_points={"console_scripts": "alphastats=alphastats.cli:run",},
         install_requires=requirements,
         extras_require=extra_requirements,
         python_requires=">=3.7,<4",
     )
 
 
 if __name__ == "__main__":
```
Comment (straussmaximilian) [inline setup.py:87]: I haven't seen a if __name__ == "__main__" in the setup.py before. Is there any reason this is needed or where it comes from?

**Review by straussmaximilian — COMMENTED** [review]
Overall, code changes look good! I made some comments for the PR, which are more high-level for polishing.
Overall, the PR was a bit larger, so feel free to make smaller PRs.

**Pull request merged**